backend/siarnaq/api/episodes/challonge.py

```diff
@@ -19,15 +20,20 @@
 URL_BASE = "https://api.challonge.com/v2/"
 
 
-def set_api_key(api_key):
-    """Set the challonge.com api credentials to use."""
-    _headers["Authorization"] = api_key
-
-
 def create_tournament(
-    tournament_url, tournament_name, is_private=True, is_single_elim=True
+    tournament_id,
```

Comment:
```suggestion
    *,
    tournament_id,
```

Force these to be kwargs so that future users don't pass them in the wrong order